arcade/utils.py

```diff
@@ -4,12 +4,29 @@
 IMPORTANT:
 These  should be standalone and not rely on any arcade imports
 """
+import functools
 import platform
 import sys
-from typing import Tuple
+import warnings
+from typing import Tuple, Type
 from pathlib import Path
 
 
+class PerformanceWarning(Type[Warning]):
+    """Use this for issuing performance warnings."""
+    pass
+
+
+def warning(message: str, warning_type: Warning):
```

Comment:
`Type[Warning]` here, sorry!

Reply:
updated in https://github.com/pythonarcade/arcade/pull/1615/commits/f7f1e15006db232aea5b33ca1c8c50edd2a8fb85